bind/map.jinja

```diff
@@ -60,7 +60,7 @@
         'mode': '640'
     },
     'FreeBSD': {
-        'pkgs': ['bind99'],
+        'pkgs': ['bind912'],
```

Comment:
According to [the ISC download page](https://www.isc.org/downloads/), 9.11 is current stable version, while 9.12 is in Early Deployment stage. Mind you change it to `bind911`?

Reply:
Good catch! Thanks!